internal/cli/atlas/backup/compliancepolicy/encryptionatrest/enable_test.go

```diff
@@ -0,0 +1,145 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build unit
+
+package encryptionatrest
+
+import (
+	"context"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli"
+	"github.com/mongodb/mongodb-atlas-cli/internal/flag"
+	"github.com/mongodb/mongodb-atlas-cli/internal/test"
+	"github.com/stretchr/testify/assert"
+	atlasv2 "go.mongodb.org/atlas-sdk/v20230201004/admin"
+)
+
+func TestEnableBuilder(t *testing.T) {
+	test.CmdValidator(
+		t,
+		EnableBuilder(),
+		0,
+		[]string{
+			flag.ProjectID,
+			flag.Output,
+			flag.EnableWatch,
+		},
+	)
+}
+
+func TestEnableOpts_InitStore(t *testing.T) {
+	opts := &EnableOpts{}
+	ctx := context.Background()
+
+	if err := opts.initStore(ctx)(); err != nil {
+		t.Fatalf("initStore()() unexpected error: %v", err)
+	}
+	assert.NotNil(t, opts.store)
+}
```

Comment:
if you gonna keep this you can simplify it for correctness
```suggestion
func TestEnableOpts_initStore(t *testing.T) {
	opts := &EnableOpts{}
	require.NoError(t, opts.initStore(context.TODO())
	assert.NotNil(t, opts.store)
}
```

Reply:
Kept it and changed test accordingly